qemu/tests/blockdev_mirror_with_ignore.py

```diff
@@ -0,0 +1,45 @@
+from avocado.utils import process
+
+from provider import job_utils
+from provider.blockdev_mirror_nowait import BlockdevMirrorNowaitTest
+
+
+class BlockdevMirrorWithIgnore(BlockdevMirrorNowaitTest):
+    """Block mirror with error ignore on target"""
+
+    def blockdev_mirror(self):
+        super(BlockdevMirrorWithIgnore, self).blockdev_mirror()
+        timeout = self.params.get("job_timeout", 600)
+        for job_id in self._jobs:
+            get_event = job_utils.get_event_by_condition
+            event = get_event(self.main_vm, job_utils.BLOCK_JOB_ERROR_EVENT,
+                              timeout, device=job_id, action='ignore')
+            if not event:
+                self.test.fail("Mirror job can't reach error after %s seconds",
```

Comment:
test.fail takes one arg, so it should be self.test.fail('xxxx %s' % timeout)

Reply:
Have updated, thanks zhenchao.